test/components/selectionBoxLayerTests.ts

```diff
@@ -118,18 +118,7 @@ describe("Interactive Components", () => {
         assert.strictEqual(sbl.xScale(), xScale, "The getter returns the correct scale");
         sbl.renderTo(svg);
 
-        let topLeft: Plottable.Point = {
-          x: 0,
-          y: 0
-        };
-        let bottomRight: Plottable.Point = {
-          x: 250,
-          y: 300
-        };
-        sbl.bounds({
-          topLeft: topLeft,
-          bottomRight: bottomRight
-        });
+        sbl.xExtent([0, 1000]);
```

Comment:
The main difference is that the unit test specifically stated that we're essentially setting the data extent of the box, not the pixel bounds of the box.